### PR TITLE
Update seeding-your-database.mdx 

### PR DESCRIPTION
**Pull request by p5quared**
## What kind of change does this PR introduce?

**Documentation Update**

## Additional Context

Updates to docs on seeding database with Snaplet.

Also see
[supabase/.../seeding-your-database](https://supabase.com/docs/guides/cli/seeding-your-database)

[snaplet/.../seed](https://docs.snaplet.dev/getting-started/quick-start/seed)

## Issue
#21089

### CODE REVIEW

**Comment by charislam** [comment]
@peterp, do you want to look this one over?

**Comment by khaya-zulu** [comment]
> @peterp, do you want to look this one over?

hi @charislam taking a look now... thanks @p5quared for opening this up.

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 1]
```diff
@@ -56,36 +56,39 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
...
-   }
-})
+const seed = await createSeedClient();
+
+await seed.posts([{
+  title: "There is a lot of snow around here!"
+  author: {
+    email: copycat.email(ctx.seed, {
+      domain: 'acme.org' 
+    }),
```
Comment (khaya-zulu) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:85]: small change.

```suggestion
    email: (ctx) =>
      copycat.email(ctx.seed, {
        domain: 'acme.org',
      })
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 2]
```diff
@@ -56,36 +56,39 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
...
+Suppose we have a database with the following schema:
 
 ![An example schema](/docs/img/guides/cli/snaplet-example-schema.png)
 
-We can use the Snaplet configuration file to define the values you want to generate. For example
+We can use the Snaplet configuration file `seed.mts` to define the values we want to generate. For example
 
 - A `Post` with the title `"There is a lot of snow around here!"`
-- The `Post.createdBy` user with an email address ending in `"@acme.org"`
+- The `Post.author` user with an email address ending in `"@acme.org"`
```
Comment (khaya-zulu) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:71]: Just looking at the schema [here](https://github.com/supabase/supabase/blob/cb70067743cd3d414fea9b5f6658b03597c1dc3a/apps/docs/public/img/guides/cli/snaplet-example-schema.png). I think its okay to keep it as `createdBy`, that way we don't have to change the image?

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 3]
```diff
@@ -56,36 +56,39 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
...
-          comments: (x) => x(3)
-        })
-      }
-   }
-})
+const seed = await createSeedClient();
+
+await seed.posts([{
+  title: "There is a lot of snow around here!"
+  author: {
```
Comment (khaya-zulu) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:82]: Then same here, instead of `author`, let's use `userCreatedBy`

**Review by khaya-zulu — CHANGES_REQUESTED** [review]
Just a few small comments, but thank you again @p5quared for pick this up for us 🙂 Let me know when you ready for another review.

File: apps/docs/pages/guides/cli/seeding-your-database.mdx (hunk 2 above)
Comment (p5quared) [reply apps/docs/pages/guides/cli/seeding-your-database.mdx:71]: Ah shoot I actually thought that and then somehow went ahead and changed them anyways; thanks!

**Review by khaya-zulu — APPROVED** [review]
Approved! thanks once again @p5quared, hope the swag gets to you 😄 I'll let @peterp know, you might have find us in our discord.

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 4]
```diff
@@ -56,36 +56,40 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
 
 You can generate seed data for local development using [Snaplet](https://www.snaplet.dev/).
 
-Let's generate some seed data for a typical blog application, with the following schema:
+If this is your first time using Snaplet to seed your project, you'll need to setup Snaplet
+for your project with `npx snaplet setup`, and follow the interactive prompts. `snaplet setup`
+will look at your database and its structure, and use it to generate the configuration files
+and assets needed by `@snaplet/seed`.
+
+Suppose we have a database with the following schema:
```
Comment (charislam) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:64]:
```suggestion
Suppose you have a database with the following schema:
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 5]
```diff
@@ -56,36 +56,40 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
...
 
 ![An example schema](/docs/img/guides/cli/snaplet-example-schema.png)
 
-We can use the Snaplet configuration file to define the values you want to generate. For example
+We can use the Snaplet configuration file `seed.mts` to define the values we want to generate. For example
```
Comment (charislam) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:68]:
```suggestion
You can use the Snaplet configuration file `seed.mts` to define the values you want to generate. For example
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 6]
```diff
@@ -56,36 +56,40 @@ COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
...
+      copycat.email(ctx.seed, {
+        domain: 'acme.org',
+      })
+  },
+  comments: (x) => x(3)
+}])
 ```
 
-Running `npx snaplet generate --sql > supabase/seed.sql` would generate these relevant SQL statements inside our `supabase/seed.sql` file:
+Running `npx tsx seed.mts > supabase/seed.sql` would generate these relevant SQL statements inside our `supabase/seed.sql` file:
```
Comment (charislam) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:92]:
```suggestion
Running `npx tsx seed.mts > supabase/seed.sql` generates the relevant SQL statements inside your `supabase/seed.sql` file:
```

**Review by charislam — APPROVED** [review]

**Pull request merged**